pkg/kubelet/config/file_linux_test.go

```diff
@@ -250,7 +223,18 @@ func (tc *testCase) writeToFile(dir, name string, t *testing.T) string {
 	return fileName
 }
 
-func watchFileAdded(watchDir bool, t *testing.T) {
+func (tc *testCase) writeSymblicLink(dir, linkedDir, name string, t *testing.T) string {
+	linkName := filepath.Join(dir, name)
+	linkedFile := filepath.Join(linkedDir, name)
+
+	err := os.Symlink(linkedFile, linkName)
+	if err != nil {
+		t.Fatalf("unexpected error when create symbolic link: %v", err)
+	}
+	return linkName
+}
+
+func watchFileAdded(watchDir bool, isSymbol bool, t *testing.T) {
```

Comment:
Does `isSymbol bool` indicate whether it's a symbolic link? `isSymlink bool` or even just `symlink bool` would be clearer.